docs/source/commandline/commandline-all.rst

```diff
@@ -17,11 +27,14 @@ Core Commands
 
 .. _mame-commandline-validate:
 
-**-validate** / **-valid**
+**-validate** / **-valid** [*<pattern>*]
 
-	Performs internal validation on every driver in the system. Run this
-	before submitting changes to ensure that you haven't violated any of
-	the core system rules.
+	Performs internal validation on one or more drivers and devices in the
+	system. Run this before submitting changes to ensure that you haven't
+	violated any of the core system rules.
+
+	If a pattern is specified, it will validate systems matching
+	the pattern, otherwise it will validate all systems.
```

Comment:
Without a pattern it validates all systems and device types (it skips validating device types if you specify a pattern).

Reply:
Fixed on my side.